Common/Orders/OrderEvent.cs

```diff
@@ -111,6 +121,25 @@ public OrderEvent(Order order, string message = "")
         }
 
         /// <summary>
+        /// Helper Constructor for TradeBuilder tests
+        /// </summary>
+        /// <param name="orderId">The Id of the order</param>
+        /// <param name="symbol">Asset Symbol</param>
+        /// <param name="time">Date/time of this event</param>
+        /// <param name="fillPrice">Fill price</param>
+        /// <param name="fillQuantity">Fill quantity</param>
+        /// <param name="orderFee">The order fee</param>
+        public OrderEvent(int orderId, string symbol, DateTime time, decimal fillPrice, int fillQuantity, decimal orderFee)
```

Comment:
Since we now require the time, I would prefer if we made it a required parameter on all constructors for the `OrderEvent` class. After that is added, this constructor can probably go away and we can define a helper method in the statistics test if need be, the reason being I don't want a developer to accidentally use this ctor which doesn't specify a status and it uses the `string symbol` instead of `Symbol symbol`